### PR TITLE
Custom Grid Line in Line Chart

### PR DESCRIPTION
**Pull request by chandrabezzo**
I just add show/hide grid line and edge line

### CODE REVIEW

File: lib/src/chart/base/axis_chart/axis_chart_data.dart [hunk 1]
```diff
@@ -25,19 +25,19 @@ abstract class AxisChartData extends BaseChartData {
...
-    this.axisTitleData,
-    this.rangeAnnotations = const RangeAnnotations(),
-    this.minX,
-    this.maxX,
-    this.minY,
-    this.maxY,
-    this.clipToBorder = false,
-    this.backgroundColor,
-  }) : super(borderData: borderData, touchData: touchData);
+  AxisChartData(
```
Comment (imaNNeoFighT) [inline lib/src/chart/base/axis_chart/axis_chart_data.dart]: Hi, please revert all reformat changes.

Comment (chandrabezzo) [reply lib/src/chart/base/axis_chart/axis_chart_data.dart]: What should I do? I just pull request, If you don't merge that I changed, you can just close without merge

Comment (imaNNeoFighT) [reply lib/src/chart/base/axis_chart/axis_chart_data.dart]: There are lot's of changes due to code formatting, they revert them and make a new commit, then I will review it.
Thanks!

**Comment by imaNNeoFighT** [comment]
It's gonna be closed tomorrow.

**Pull request closed without merging**